CHANGES/8510.misc.rst

```diff
@@ -0,0 +1 @@
+Avoid scheduling the writer if it can finish synchronously -- by :user:`bdraco`.
```

Comment:
Could you change this from the imperative TODO-list instruction style to something that refers to what's changed since the previous release + what noticeable effect the end-users would observe?

Reply:
Adjusted in 74718c1c